config/lib/express.js

```diff
@@ -250,51 +241,6 @@ module.exports.initModulesConfiguration = function (app, db) {
  * https://helmetjs.github.io/docs/
  */
 module.exports.initHelmetHeaders = function (app) {
-  /**
-   * X-Frame protection ("frameguard") default options.
-   * @link https://helmetjs.github.io/docs/frameguard/
-   */
-  let frameguardOptions = {
-    // Action `sameorigin` will prevent anyone from putting this page in an
-    // iframe unless it’s on the same origin. That generally means that you can
-    // put your own pages in iframes, but nobody else can.
-    //
-    // action `deny` will prevent anyone from putting this page in an iframe.
-    action: 'sameorigin',
-  };
-
-  /**
-   * Content Security Policy (CSP) "frameAncestors" default value.
-   * @link https://helmetjs.github.io/docs/csp/
-   */
-  let cspFrameAncestors = ["'none'"];
-
-  /**
-   * If Facebook notifications are enabled, override default options for:
-   * - X-Frame protection
-   * - Content Security Policy (CSP) "frameAncestors" value
-   *
-   * This is required for FB canvas to work,
-   * which in turn is required for FB notifications only.
-   *
-   * X-Frame protection's `allow-from` will allow `https://apps.facebook.com`
-   * to put your page in an iframe, but nobody else.
-   * Unfortunately, you can only allow one domain and this doesn't work
-   * with Chrome as they use CSP FrameAncestors instead.
-   *
-   * @link https://helmetjs.github.io/docs/frameguard/
-   * @link https://canvas.facebook.com
-   * @link https://developers.facebook.com/docs/games/gamesonfacebook
-   * @link https://developers.facebook.com/docs/games/services/appnotifications
-   */
-  if (facebookNotificationService.isNotificationsEnabled()) {
-    frameguardOptions = {
-      action: 'allow-from',
```

Comment:
This action has been deprecated, hence I'm removing it.

https://github.com/helmetjs/helmet/wiki/How-to-use-X%E2%80%93Frame%E2%80%93Options's-%60ALLOW%E2%80%93FROM%60-directive